### PR TITLE
Properly translate from SerialError to lin_bus::Error

### PR DESCRIPTION
**Pull request by rnestler**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+## [Unreleased]
+
+ * changed: Properly translate serial errors to lin bus errors
+   ([#6](https://github.com/Sensirion/lin-bus-driver-serial-rs/pull/6))
+
+## 0.1.0 (2018-06-25)
+
+ * First crates.io release
+
+[Unreleased]: https://github.com/Sensirion/lin-bus-driver-serial-rs/compare/v0.1.0...HEAD
```
Comment (abrauchli) [inline CHANGELOG.md:14]: Since there's an unreleased header above 0.1.0, did you intend for this to be below the 0.1.0 header?

**Review by abrauchli — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (rnestler) [reply CHANGELOG.md:14]: What do you mean? This is just the link target for the Unreleased header so you can see what differences are unreleased.

Comment (rnestler) [reply CHANGELOG.md:14]: See https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links "reference style links"

Comment (abrauchli) [reply CHANGELOG.md:14]: Thanks, I didn't know about that. It's clear when you see it rendered: https://github.com/Sensirion/lin-bus-driver-serial-rs/blob/d9af0004f82dad5242b4c9637d2daed3fb75d845/CHANGELOG.md - should have done that in the first place.

**Pull request merged**